rotkehlchen/chain/gnosis/modules/sdai/constants.py

```diff
@@ -0,0 +1,7 @@
+from typing import Final
+from rotkehlchen.chain.evm.types import string_to_evm_address
+
+CPT_SDAI = 'sDAI'
+GNOSIS_SDAI_ADDRESS: Final = string_to_evm_address('0xaf204776c7245bF4147c2612BF6e5972Ee483701')
+SDAI_LABEL = 'sDAI contract'
+SDAI_ICON = 'sdai.svg'
```

Comment:
ask in discord if you need help but you can also add the icon where it is needed. You can check my eigenlayer PR to see where it is needed https://github.com/rotki/rotki/pull/7410